org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/AbstractAcquisitionTest.java

```diff
@@ -233,7 +234,8 @@ protected IDeviceController createTestScanner(IScannable<?> monitor) throws Exce
 		
 		if (device==null) device = (IRunnableDevice<T>)detector;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "device". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1226)